schema.graphql

```diff
@@ -89,3 +89,10 @@ type SystemTokenTransfer @entity {
   timestamp: Date @index
   extrinsic: Extrinsic
 }
+
+type VestingSchedule @entity {
+  id: ID!
+  signer: String @index
+  to: String
+  data: String
```

Comment:
Can you better define the type of the `data` field here? The structure that we have for the data part of a VestingSchedule transaction should look like:
```
{
  start: Int
  period: Int
  periodCount: Int
  PerPeriod: BigInt
}
```
You can possibly remove `data` and just have the above field.

Reply:
Yeah. Currently I have used this field as a string. and exchange this field type on web-wallet.
I will change the field type now.